presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionTask.java

```diff
@@ -56,6 +58,8 @@
 public class NativeExecutionTask
 {
     private static final Logger log = Logger.get(NativeExecutionTask.class);
+    // TODO: add to config
```

Comment:
Should we address this TODO now?

Any particular reason not to use existing config?

query.remote-task.max-error-duration